demo/demo-tekton

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+#Assumes User is logged in to cluster
+set -euo pipefail
+
+all_cmds_found=0
+for cmd in oc tkn; do
+   if ! which -s $cmd; then
+      echo "No $cmd executable found in PATH" >&2
+      all_cmds_found=1
+   fi
+done
+if ! [[ $all_cmds_found ]]; then exit 1; fi
+
+
+tekton_setup_dir="$(dirname $0)/tekton-demo-setup"
```

Comment:
If we are sure it's bash, then it's better to use:
$(dirname "$BASH_SOURCE")

In other case use:
SCRIPT_PATH=$(readlink "$0")
$(dirname "$SCRIPT_PATH")